@pauliescanlon/gatsby-mdx-embed/src/components/Egghead/Egghead.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react'
+import { getPadding } from '../../utils'
+import { GeneralObserver } from '../GeneralObserver'
+
+export interface IEggheadProps {
+  /** Egghead lesson */
+  lesson: string
```

Comment:
Do you think this should be `lessonName` or `lessonId` perhaps. Or would using some other word other than lesson make more sense. Not sure how this corresponds to what you're looking at on egghead. How do egghead refer to a video?

Reply:
🤔 I know they use the term course for courses, but not sure about individual videos??

Amma check now!

Reply:
They're:

- lesson
- collection
- podcast
- course

So I'd `lessonId`, ok?

Reply:
Yeah `lessonId` works but.... Just curious. Is it possible to embed collections and podcast or courses?

Reason i ask is that this component is called Egghead but it suppose if you could emebed postcasts what would that component be called. 

A bit like with Twitter, there's no Twitter component, it's Tweet, TwitterFollowButton etc, same deal with Pinterest... know what i mean

Reply:
Ah! Gotcha! I've not seen anything other than videos, maybe need to spend a bit more time on this one then??

Reply:
it might be worth investigating a little. If we publish an `Egghead` component we might need to deprecate it later if it's possible to do `EggheadPodcast` in which case it might become `EggheadLesson` etc.. just trying to future proof it and not have to publish it as a breaking change. This still isn't even a v1 and i was hoping to keep it that way for a little while longer until i've got proper feedback about it.... but it needs to become famous first 😂